src/MLJInterface.jl

```diff
@@ -167,18 +168,20 @@ MLJModelInterface.@mlj_model mutable struct LGBMClassifier <: MLJModelInterface.
     device_type::String = "cpu"::(_ in ("cpu", "gpu"))
     force_col_wise::Bool = false
     force_row_wise::Bool = false
+    truncate_booster::Bool = true
 
 end
 
 
 MODELS = Union{LGBMClassifier, LGBMRegressor}
 
 
-function mlj_to_kwargs(model::LGBMRegressor)
+function mlj_to_kwargs(model::MODELS)
 
     return Dict{Symbol, Any}(
         name => getfield(model, name)
         for name in fieldnames(typeof(model))
+        if name != :truncate_booster
```

Comment:
We might end up adding a few more things here. Maybe get a constant somewhere to keep track of these things?